tests/data/urban3d/data.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+import random
+
+import rasterio
+import numpy as np
+
+from torchgeo.datasets import Urban3DChallenge
+
+ROOT = "/path/to/dataset/"
+NUM_SAMPLES = 2
+SIZE = 32
+
+np.random.seed(0)
+random.seed(0)
+
+
+def create_file(path: str, output_path: str) -> None:
+    src = rasterio.open(path)
```

Comment:
Can you change this script to hardcode the profile instead of relying on already-downloaded data?